scripts/unique_fuzz_errors.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Nit: rename file to `fuzz_unique_errors.sh` (as a convention, we use common prefixes to group files, types, functions, variables, etc).